packages/amazonq/package.json

```diff
@@ -110,6 +110,10 @@
                         "amazonQSessionConfigurationMessage": {
                             "type": "boolean",
                             "default": false
+                        },
+                        "ssoCacheError": {
```

Comment:
These are just settings to allow to user to suppress the file system error message